tests/fixtures/test-protobuf-schema.proto

```
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package testdata;
```

Buf (1.64.0)

[error] Files with package "testdata" must be within a directory "testdata" relative to root but were in directory "tests/fixtures".

(PACKAGE_DIRECTORY_MATCH)